server/routerlicious/packages/routerlicious-base/src/utils/constants.ts

```diff
@@ -7,4 +7,6 @@ export const Constants = Object.freeze({
 	alfredRestThrottleIdSuffix: "AlfredRest",
```

Comment:
can we make this lowerCase as well?

Reply:
We all set these as upper case, such as HistorianRest, AlfredRest, and EchoRest. :)